### PR TITLE
Add ROS!

### PR DESCRIPTION
**Pull request by da-luce**
Begin using ROS as the basis for our project.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,13 +1,32 @@
 # Cache files
 __pycache__/
 *cache*
+<<<<<<< HEAD
```
Comment (AriMirsky) [inline .gitignore]: This looks like an unresolved merge conflict?

File: pyproject.toml [hunk 2]
```diff
@@ -41,17 +41,22 @@ dependencies = [
...
+pythonpath = ["src"]
 
 [tool.black]
 line-length = 88
 target-version = ["py39"]
 skip-string-normalization = true
 
 [tool.pylint]
-disable = ["W0511", "C0301"]
+disable = ["W0511", "C0301", "E1101", "E0237"]
```
Comment (AriMirsky) [inline pyproject.toml:54]: Just wondering, why are we disabling these rules?

**Review by AriMirsky — CHANGES_REQUESTED** [review]
I'm also wondering what's your plan with the development process for the unrosified folder? Besides that, just fixing the gitignore is the only reason I didn't approve this pr

**Comment by da-luce** [comment]
@AriMirsky I was planning on using whatever we can from unrosified to ROS nodes as we find fit. Nothing really specific, just to reuse and refactor as necessary.

File: pyproject.toml (hunk 2 above)
Comment (da-luce) [reply pyproject.toml:54]: The way ROS requires us to setup the code base results in a few linting errors within Python. It's rather annoying and couldn't find a reasonable solution so this is like a short-term solution to get linting to pass.

File: .gitignore (hunk 1 above)
Comment (da-luce) [reply .gitignore]: Oops, will take care of that

**Review by AriMirsky — APPROVED** [review]

**Pull request merged**